v03_pipeline/bin/pipeline_worker.py

```diff
@@ -73,6 +76,14 @@ def main():
                     for i in range(len(lpr.projects_to_run))
                 ],
             ]
+            if Env.SHOULD_TRIGGER_HAIL_BACKEND_RELOAD:
+                tasks.append(
+                    TriggerHailBackendReload(
```

Comment:
I think we need something that'll make sure this task runs after all the others.  Maybe `require` the WriteProjectFamilyTablesTask?

Reply:
Or even better, the annotations table task, since the `/reload` route very specifically has to do with the globals on that table!

Reply:
Ok, I made `UpdateVariantAnnotationsTableWithNewSamplesTask` a requirement of the new task. 

I introduced another class `BaseProjectInfoParams` to track the project-related parameters instead of copy pasting them into the new task.

Reply:
Yep that sounds like the right thing for sure!